tests/testsBuilder.cpp

```diff
@@ -95,6 +95,13 @@ TEST(BuilderTest, AddObjectIteratorKeyAlreadyWritten) {
   ASSERT_FALSE(b.isClosed());
 }
 
+TEST(BuilderTest, AddToObjectEmtpyStringView) {
+  Builder obj;
+  obj.openObject();
+  obj.add(std::string_view{}, std::string_view{});
+  obj.close();
```

Comment:
I guess this "test" is only here to check if UBSan warns at runtime. We should then at least leave a comment in this test case, because otherwise it is not obvious what its purpose is (there are no `ASSERT`s or `EXPECT`s here).

Reply:
This test check velocypack assert

Reply:
You can add it to master and see it fails

Reply:
What are your questions? If the changes are necessary for velocypack type `Binary` as well?
I think yes, even if it may not be called with a nullptr there, the changes don't cause any harm.

Reply:
Yes. Hmm ok, what is about slice?
My question: Is it possible for binary/slice to have size == 0?

Reply:
> Yes. Hmm ok, what is about slice? My question: Is it possible for binary/slice to have size == 0?

A velocypack `Binary`'s content can have a size of zero.

Reply:
I don't know how add binary as key and value :(

And still have question about slice.
it called when:
```cpp
Object o;
o.open();
o.add(Slice)
o.close();
```

Reply:
Pushed a change into the PR for extending the test.